pontoon/api/schema.py

```diff
@@ -112,6 +115,54 @@ def resolve_localizations(obj, _info, include_disabled):
         return qs.filter(project__disabled=False)
 
 
+class ActionLog(DjangoObjectType):
+    class Meta:
+        model = ActionLogModel
+        only_fields = (
+            "action_type",
+            "created_at",
+            "performed_by",
+            "translation",
+        )
+
+
+class User(DjangoObjectType):
+    class Meta:
+        convert_choices_to_enum = False
+        model = UserModel
+        only_fields = (
+            "username",
+            "first_name",
+            "last_name",
+        )
+
+    actions = graphene.List(ActionLog)
+
+    def resolve_actions(obj, info):
+        return ActionLogModel.objects.filter(performed_by=obj)
+
+
+class Translation(DjangoObjectType):
+    class Meta:
+        model = TranslationModel
+        only_fields = (
+            "user",
+            "locale",
```

Comment:
Do we need to expose `locale` here? It seems like the only way to query for translations is with `project` and `locale` parameters anyways.

Reply:
Ah, no, that's wrong. It's possible to do `actions.translation` from the root, from `translations`, and from `user`. Is it needed?